src/silo/preprocessing/preprocessor.cpp

```diff
@@ -555,6 +555,24 @@ void Preprocessor::createUnalignedPartitionedSequenceFile(
    const std::filesystem::path save_location =
       preprocessing_config.getIntermediateResultsDirectory() /
       fmt::format("unaligned_nuc_{}", sequence_idx);
+   // duckdb OVERWRITE and OVERWRITE_OR_IGNORE is broken in the current version,
```

Comment:
It would be very cool if you could:
1. add the link to the duckdb issue in a code comment here, or if none exists open one
2. include a link to the duckdb issue in the PR description
3. explicitly state which is the current duckdb version, and since approximately which duckdb version this bug is present in duckdb

Reply:
Why don't we we pin duckdb to the highest non-bugged version?

Reply:
I added some info, including the link to the [change log](https://github.com/GenSpectrum/LAPIS-SILO/blob/main/CHANGELOG.md). Unfortunately, I saw that it did not appear in the release notes. Updating duckdb is not always trivial, esp. in the versions before 1.0.0 the build script often changed and the recipes in the package manager we use (conan) needed to be changed [multiple](https://github.com/conan-io/conan-center-index/commit/ba3fe9f0ee53b80620e74acdab461c283d03a145) [times](https://github.com/conan-io/conan-center-index/commit/0b2998d329462c48b561abfdd2b6b9f155a8f58a), given that we also use some duckdb extensions, which the maintainer of that recipe often does not check...

The most recent update was delayed, because I thought that #394 would make some of these migrations more deterministic, given that the query results of unsorted or randomly sampled queries often changed.